firefox-ios/Shared/Strings.swift

```diff
@@ -2055,7 +2055,61 @@ extension String {
             value: "Summarizing…",
             comment: "When the user uses the summarizing feature, this is the loading label that is shown while the summarization is being performed."
         )
+        public static let ToSAlertTitleLabel = MZLocalizedString(
+            key: "Summarizer.ToS.Alert.Title.Label.v142",
+            tableName: "Summarizer",
+            value: "Allow Page Summaries?",
+            comment: "The title for the ToS alert that asks the user if they want to allow page summaries."
+        )
+        public static let ToSAlertMessageFirefoxLabel = MZLocalizedString(
+            key: "Summarizer.ToS.Alert.FirefoxMessage.Label.v142",
+            tableName: "Summarizer",
+            value: "Firefox uses AI securely hosted on Mozilla servers to summarize pages.",
```

Comment:
Firefox and Mozilla should be replaced by placeholders so that we make sure they won't get translated.